[minor] Support generation of BYO JDBC config

sbudhira opened the pull request:
Added JDBC Cfg for external db. 
- JDBC cfg will be created and deployed
- All three DBs are supported for Manage (DB2, Oracle and SQL Server)
- Workspace-application scoped JDBC Cfg will be created for Manage app.
- System scoped JDBC Cfg will be created for IoT app.

CLI Sample
```
7.1 Database configuration for IoT
Maximo IoT requires a shared system-scope Db2 instance because others application in the suite require access to the same database source.
 - Only IBM Db2 is supported for this database

System Database configuration for IoT is not required because the application is not being installed

7.2 Database configuration for Manage
Maximo Manage can be configured to share the system Db2 instance or use it's own dedicated database.
 - IBM Db2, Oracle Database, & Microsoft SQL Server are all supported database options

Do you want to generate a dedicated JDBC configuration for Manage? [y/N] y

Select Local configuration directory > /mascli/masconfig
Configuration Display Name: jdbc-sb1-manage
JDBC Connection String: jdbc:sqlserver://;serverName=ssldbsvl1.fyre.ibm.com;portNumber=1433;databaseName=maxdb80;integratedSecurity=false;sendStringParametersAsUnicode=false;selectMethod=cursor;encrypt=true;trustServerCertificate=true;
JDBC User Name: maximo
JDBC Password: myStrongPassword9
SSL Enabled [y/n]: y
Path to certificate file: /mascli/masconfig/mssql.pem
Configuring workspace-application JDBC configuration for sb1

8. Additional Configuration
Additional resource definitions can be applied to the OpenShift Cluster during the MAS configuration step.
The primary purpose of this is to apply configuration for Maximo Application Suite itself, but you can use this to deploy ANY additional resource into your cluster.

The following additional configurations will be applied:
 - jdbc-sb1-wsapp.yaml

Are these the correct configuration files to apply? [y/N] y
```
[jdbccfg-sb1-jdbc-wsapp-masdev-manage.zip](https://github.com/ibm-mas/cli/files/11058766/jdbccfg-sb1-jdbc-wsapp-masdev-manage.zip)

durera committed:
[minor] Support for MAS 8.10 SUDS-based install

durera committed:
[minor] Prompt the user to generate a jdbccfg

Allow user to generate a JDBC configuration on the fly during mas install command in cases where they want to use an external database and do not have a pre-generated configuration file.

This system can be extended to support all other MAS *cfg resource definition generation

sbudhira committed:
[minor] add jdccfg file parameter

sbudhira committed:
[minor] add jdccfg file parameter

sbudhira committed:
[minor] add jdccfg for external db

sbudhira committed:
[minor] add jdccfg for external db

durera committed:
Various fixes

durera committed:
Use click.Path instead of str for arg

sbudhira committed:
[minor] add jdccfg for external db

sbudhira committed:
[minor] add jdccfg for external db

sbudhira committed:
[minor] add jdccfg for external db

sbudhira committed:
[minor] add jdccfg for external db

sbudhira committed:
[minor] add jdccfg for external db

sbudhira committed:
[minor] add jdccfg for external db

sbudhira committed:
[minor] add jdccfg for external db

sbudhira committed:
[minor] add jdccfg for external db

sbudhira committed:
[minor] add jdccfg for external db

sbudhira committed:
[minor] add jdccfg for external db

sbudhira committed:
Merge branch 'master' into jdbccfg

durera commented on image/cli/mascli/functions/pipeline_config_additional_configs:
```diff
@@ -28,7 +37,9 @@ function config_pipeline_additional_configs() {
-    $DIR/tools/cfgmgr add $CONFIG_DIR/additional-configs-$MAS_INSTANCE_ID.yaml $LOCAL_MAS_CONFIG_DIR
+    cp $DIR/templates/additional-configs.yaml $CONFIG_DIR/additional-configs-$MAS_INSTANCE_ID.yaml
+    python3 $DIR/tools/cfgmgr add $CONFIG_DIR/additional-configs-$MAS_INSTANCE_ID.yaml $LOCAL_MAS_CONFIG_DIR
+    CONFIGURE_EXTERNAL_DB="true"
```
Why was `CONFIGURE_EXTERNAL_DB` added?  It's serving no value, we already have `db2_action_manage` that tells us whether or not we are setting up a database for manage, this appears to be completely redundant to me.

durera replied:
Fixed it myself

durera committed:
Remove CONFIGURE_EXTERNAL_DB

We already have parameters that serve this purpose, the stages in the pipeline will skip unless the appropriate db_action is set to "install"

durera committed:
Put the stage numbers back to correct ones

durera committed:
Need to undo changes that broke SSL enabled support

durera committed:
Clean up boilerplate code left behind

durera committed:
Remove prompt for optional ssl-cert option
